initializr-generator/src/main/java/io/spring/initializr/util/VersionRange.java

```diff
@@ -105,6 +105,22 @@ public boolean isHigherInclusive() {
 		return this.higherInclusive;
 	}
 
+	public String toVersionRange() {
```

Comment:
I find it a bit confusing having `toVersionRange()` (that returns a `String`) on a `VersionRange` class. `toString` is already taken. Perhaps it could be called `toRangeString()` instead?

Reply:
agreed, it is quite confusing. I had it as `toVersionRange` because the String field on `Dependency` is called `versionRange` but `toRangeString` sounds good to me.